helfi_platform_config.module

```diff
@@ -467,3 +467,24 @@ function helfi_platform_config_language_switch_links_alter(array &$links): void
     $links[$langcode]['attributes']['class'][] = 'language-link--untranslated';
   }
 }
+
+/**
+ * Implements hook_config_ignore_settings_alter().
+ */
+function helfi_platform_config_config_ignore_settings_alter(array &$settings) {
+  $add_ignore = [
+    'block.block.ibmchatapp_*',
```

Comment:
I think `block.block.ibmchatapp` should be ignored as well.

Sote has chat block with this id: https://github.com/City-of-Helsinki/drupal-helfi-sote/blob/dev/conf/cmi/block.block.ibmchatapp.yml